### DISCUSSION TITLE
@babel/eslint-parser versions going backwards

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#20967] [Suggest an Idea]
### Type of discussion.

I just want to chat

### Tell us more.

@travi just wanted to check if PRs like this were intended and ok? https://github.com/elenabravo/simple-react-starter/pull/503/files

It comes from https://github.com/renovatebot/renovate/pull/13277 which we now made default behavior in v35.

Related links:
https://www.npmjs.com/package/babel-eslint
https://www.npmjs.com/package/@babel/eslint-parser

**Comment by travi** [comment]
It's been a bit, so I needed to take a look again. From what I can see, the versions between the two packages are completely independent. The replacement version is the first public version from the new package and the release timing aligned with the final version of the old package.
